tests/componenttests/server/tests/mediaPipelineCapabilities/MediaPipelineCapabilitiesTest.cpp

```diff
@@ -35,6 +35,7 @@ constexpr int kNumPropertiesOnSink{3};
 const char *kPropertyName1 = "test-name-5";
 const char *kPropertyName2 = "test2";
 const char *kPropertyName3 = "prop";
+const char *kPropertyName4 = "audio-fade";
```

Comment:
I would name this constant kAudioFadePropertyName. The others are generic but this one is not.